### ISSUE TITLE
a lot of errors when trying to compile for Feather M0 Express

### ISSUE THREAD

**Issue by Impact9** [adafruit/Adafruit_Learning_System_Guides#456]
Adafruit is unclear with exactly how to perform this project. I'm trying to compile this for my Adafruit Feather M0 Express Hallowing but I'm finding a lot of errors and being a novice I'm not sure what's wrong, me or the code. Following this guide (https://learn.adafruit.com/hallowing-all-seeing-skull/code-in-arduino) to here. 

I did load the libraries and boards instructed but the only thing that wasn't clear was the Programmer. I've tried AVRISP MK II with the following results on Arduino 1.8.8 - 

`Arduino: 1.8.8 (Windows 10), Board: "Adafruit Feather M0 Express"

All_Seeing_Skull:290:7: error: stray '\342' in program

       <span aria-hidden="true" class="d-inline-block ml-1 v-align-middle">â†µ</span>

       ^

**Comment by ladyada** [comment]
hi please go back to the giude and click this button, you shoudl get a zip

![image](https://user-images.githubusercontent.com/1214161/50551844-b6321600-0c55-11e9-8236-6ae62fcae412.png)

**Comment by Impact9** [comment]
I'm not sure why but it compiled and uploaded this time but it **will not** work. It did initially work when I first powered it up but I was wanting to change eyes which caused this problem. Now I did have to delete the Arduino15 folders that were hidden in my user folder to get this to compile. I have both the Arduino and Adafruit SAMD along with the Adafruit GFX, ST7735/ST7789, and Zero DMA included. I just can't get it to work. I shouldn't need to do anything else beyond commenting out the OLED lines in the config.ch right?
